From 1828608a2057898fdde8198d47a9ce7b50839369 Mon Sep 17 00:00:00 2001
From: Kunhui-Li <kunhuix.li@intel.com>
Date: Fri, 23 Jul 2021 10:25:24 +0800
Subject: [PATCH 4/4] config-tools: add uclock GP switch

A switch to force disable GP for UC lock using scenario configuration.

Tracked-On: #6299
Signed-off-by: Kunhui-Li <kunhuix.li@intel.com>
---
 misc/config_tools/data/adl-s-crb/hybrid.xml            | 1 +
 misc/config_tools/data/adl-s-crb/hybrid_rt.xml         | 1 +
 misc/config_tools/data/adl-s-crb/industry.xml          | 1 +
 misc/config_tools/data/adl-s-crb/logical_partition.xml | 1 +
 4 files changed, 4 insertions(+)

diff --git a/misc/config_tools/data/adl-s-crb/hybrid.xml b/misc/config_tools/data/adl-s-crb/hybrid.xml
index f4e67ec3c..7d2bf1a26 100644
--- a/misc/config_tools/data/adl-s-crb/hybrid.xml
+++ b/misc/config_tools/data/adl-s-crb/hybrid.xml
@@ -15,6 +15,7 @@
             <SCHEDULER>SCHED_BVT</SCHEDULER>
             <MULTIBOOT2>y</MULTIBOOT2>
             <ENFORCE_TURNOFF_AC>y</ENFORCE_TURNOFF_AC>
+            <ENFORCE_TURNOFF_GP>n</ENFORCE_TURNOFF_GP>
             <RDT>
                 <RDT_ENABLED>n</RDT_ENABLED>
                 <CDP_ENABLED>n</CDP_ENABLED>
diff --git a/misc/config_tools/data/adl-s-crb/hybrid_rt.xml b/misc/config_tools/data/adl-s-crb/hybrid_rt.xml
index c2afddd62..57aeedb5f 100644
--- a/misc/config_tools/data/adl-s-crb/hybrid_rt.xml
+++ b/misc/config_tools/data/adl-s-crb/hybrid_rt.xml
@@ -14,6 +14,7 @@
       <SCHEDULER>SCHED_BVT</SCHEDULER>
       <MULTIBOOT2>y</MULTIBOOT2>
       <ENFORCE_TURNOFF_AC>y</ENFORCE_TURNOFF_AC>
+      <ENFORCE_TURNOFF_GP>n</ENFORCE_TURNOFF_GP>
       <RDT>
         <RDT_ENABLED>n</RDT_ENABLED>
         <CDP_ENABLED>y</CDP_ENABLED>
diff --git a/misc/config_tools/data/adl-s-crb/industry.xml b/misc/config_tools/data/adl-s-crb/industry.xml
index a3d8f37d6..bb9434f88 100644
--- a/misc/config_tools/data/adl-s-crb/industry.xml
+++ b/misc/config_tools/data/adl-s-crb/industry.xml
@@ -14,6 +14,7 @@
       <SCHEDULER>SCHED_BVT</SCHEDULER>
       <MULTIBOOT2>y</MULTIBOOT2>
       <ENFORCE_TURNOFF_AC>y</ENFORCE_TURNOFF_AC>
+      <ENFORCE_TURNOFF_GP>n</ENFORCE_TURNOFF_GP>
       <RDT>
         <RDT_ENABLED>n</RDT_ENABLED>
         <CDP_ENABLED>y</CDP_ENABLED>
diff --git a/misc/config_tools/data/adl-s-crb/logical_partition.xml b/misc/config_tools/data/adl-s-crb/logical_partition.xml
index b83c4c0c4..8dd093180 100644
--- a/misc/config_tools/data/adl-s-crb/logical_partition.xml
+++ b/misc/config_tools/data/adl-s-crb/logical_partition.xml
@@ -14,6 +14,7 @@
       <SCHEDULER>SCHED_BVT</SCHEDULER>
       <MULTIBOOT2>y</MULTIBOOT2>
       <ENFORCE_TURNOFF_AC>y</ENFORCE_TURNOFF_AC>
+      <ENFORCE_TURNOFF_GP>n</ENFORCE_TURNOFF_GP>
       <RDT>
         <RDT_ENABLED>n</RDT_ENABLED>
         <CDP_ENABLED>n</CDP_ENABLED>
-- 
2.25.1

